src/ERC5564Announcer.sol

```diff
@@ -1,5 +1,5 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity 0.8.20;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
 
 import {IERC5564Announcer} from "./interfaces/IERC5564Announcer.sol";
```

Comment:
Just noticed this contract is still inheriting from `IERC5564Announcer`. Like the registry, it should not include any imports and should not need to inherit from that interface.

Reply:
Removed the import here [0c32536](https://github.com/ScopeLift/stealth-address-erc-contracts/pull/5/commits/0c32536de2ed267d8760f98eb70cbbcbcd2967a5)